tbx/services/blocks.py

```diff
@@ -14,11 +14,38 @@
 from wagtail.images.blocks import ImageChooserBlock
 
 
+class PartnerLogoImageChooserBlock(ImageChooserBlock):
```

Comment:
this could also use the generic ImageChooser class that prefetches all renditions